src/components/Form/styled.ts

```diff
@@ -112,3 +112,13 @@ export const FormIcon = styled.div`
 		cursor: pointer;
 	}
 `
+export const FormTextBlock = styled.textarea`
+	box-sizing: border-box;
+	width: 95%;
+	margin: auto;
+	height: 12em;
```

Comment:
Is text area always 12em in height? :thinking: If we only have around 3 rows of text allowed (maybe limit it with number of letters?), then it's confusing. User thinks he can add this big 12em paragraph but instead he's allowed only few sentences. 

In the end with this, avatar is too big (get blurry, recommended avatar width/height is at max. 200px, this is almost 250px) and after submitting motto, we get huge "gaps" between nickname and motto when it's short.

> Default velikosti souboru je něco kolem 200kb nebo velikostně něco jako 120 - 200px.

![avatar2](https://user-images.githubusercontent.com/76131194/111761411-26153100-88a0-11eb-9524-d839d78c5754.png)

Reply:
Textarea has no restrictions yet and 12em is high due to the "Something about me" section. I can set other styles for the motto and other styles elsewhere if we really want to.

I don't understand the second paragraph very well 🤔 I don't understand the second paragraph very well. I reduce the image to 200px and I don't understand the gap.

Reply:
This is what I meant. In current case, each gap (margin or padding) is different. size. 

Name is pinned down:
![current one](https://user-images.githubusercontent.com/76131194/111777859-261f2c00-88b4-11eb-82fb-d6ff4caaf4e1.png)

While motto is centered:
![centered](https://user-images.githubusercontent.com/76131194/111777957-4353fa80-88b4-11eb-8b84-bb7eb85a73ba.png)

Resulting in three different gaps:
![každájiná](https://user-images.githubusercontent.com/76131194/111777665-e0626380-88b3-11eb-9deb-30f01c5191e5.png)

While this is what happens if you don't vertically center motto:
![somethinglikethis](https://user-images.githubusercontent.com/76131194/111777670-e1939080-88b3-11eb-9855-aee3f94dbf24.png)

Reply:
Well, we have to restrict this (Props?) or it'll overflow. Allowing users to put there any ammount of text completely destroys the idea of "**short first impression** motto". Making it short is also necessary for it to fit next to the avatar without breaking the design at some point. 

Gap explained in comment about centering. It should be horizontally centered (left and right), I didn't like how middle vertical gap centeres it (also in another comment with screenshots).